system-tests/projects/svelte-vite-unconfigured/.gitignore

```diff
@@ -0,0 +1,24 @@
+# Logs
```

Comment:
Do we need this? I think all of these are ignored in the top level `.gitignore`?